### PR TITLE
Change image name to ez-rancher

### PR DESCRIPTION
**Pull request by hulquest**
Added a secret for docker hub and linted the README.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,20 +1,27 @@
 IMAGE_TAG ?= dev
-REGISTRY ?= index.docker.io/netapp # Can substitute gcr, quay or registry:5000
+IMAGE_NAME ?= ez-rancher
```
Comment (bgentry-ntap) [inline Makefile:2]: Is there any use case to change the image name?

File: hack/runner.sh [hunk 2]
```diff
@@ -41,10 +46,11 @@ fi
 if [ "$OPERATION" == "destroy" ]; then
   # Remove Rancher resources from project state prior to running destroy. 
   # This will allow any additional clusters that the user has created to be retained.
-  docker run -it --rm -v "$TFVARS":/terraform/vsphere-rancher/rancher.tfvars -v "$DELIVERABLES":/terraform/vsphere-rancher/deliverables terraform-rancher:"$IMAGE_TAG" state rm module.rancher
-  docker run -it --rm -v "$TFVARS":/terraform/vsphere-rancher/rancher.tfvars -v "$DELIVERABLES":/terraform/vsphere-rancher/deliverables terraform-rancher:"$IMAGE_TAG" "$OPERATION" -auto-approve -var-file=/terraform/vsphere-rancher/rancher.tfvars -target=module.cluster_nodes.vsphere_virtual_machine.node
+  docker run -it --rm -v "$TFVARS":/terraform/vsphere-rancher/rancher.tfvars -v "$DELIVERABLES":/terraform/vsphere-rancher/deliverables ez-rancher:"$IMAGE_TAG" state rm module.rancher
```
Comment (bgentry-ntap) [inline hack/runner.sh:49]: I think we want the registry included here

```
$  ./hack/runner.sh destroy

Unable to find image 'ez-rancher:latest' locally
docker: Error response from daemon: pull access denied for ez-rancher, repository does not exist or may require 'docker login'.
```

File: README.md [hunk 3]
```diff
@@ -2,15 +2,17 @@
 
 ----
 
-Terraform Rancher is Infrastructure as Code to deploy Rancher server on vSphere. It creates the VM infrastructure and configures [Rancher server](https://rancher.com/docs/rancher/v2.x/en/overview/). 
+Terraform Rancher is Infrastructure as Code to deploy Rancher server on vSphere. It creates the VM infrastructure and configures [Rancher server](https://rancher.com/docs/rancher/v2.x/en/overview/).
```
Comment (bgentry-ntap) [inline README.md:5]: We should probably change all references of 'Terraform Rancher' to ez-rancher as well

File: hack/runner.sh (hunk 2 above)
Comment (hulquest) [reply hack/runner.sh]: You're right.  Merge error that's been fixed.

**Pull request merged**